### PR TITLE
2.3.0 - version bump, header updates, readme update

### PR DESCRIPTION
**Pull request by jhudsonWA**
Bump verison to 2.3.0, move headers in example projects to use ZipArchive.h headers and update readme to include SPM

### CODE REVIEW

**Comment by jhudsonWA** [comment]
@Coeur I think a version bump would be helpful. While it's not required for SPM (they can reference master), it helps if we have a tag that includes the manifest for version-based updates. 

I also updated the readme to include SPM (and added imports to the code examples).

Please review and merge in if it looks good. 

Next I'll bump minizip version for a 2.3.0, but I want to get the last year's changes out in some format.

**Comment by Coeur** [comment]
Don't hesitate to bump to 2.3.0 instead of 2.2.4.

File: README.md [hunk 1]
```diff
@@ -58,6 +65,9 @@ SSZipArchive requires ARC.
 ### Swift
 
 ```swift
+//Import
+import ZipArchive //import SSZipArchive still works for previous code
```
Comment (Coeur) [inline README.md:69]: Are you sure that both import ZipArchive and import SSZipArchive will work with our latest version?

Comment (jhudsonWA) [reply README.md:69]: So best I can tell, is it won't work if you pull it in via SPM, but if you had SSZipArchive before via another method it works. I tried both and both build, but then again, both have always built for me.

Comment (jhudsonWA) [reply README.md:69]: Another option would be to just announce a possible breaking change and push towards ZipArchive for all. Thoughts?

Comment (Coeur) [reply README.md:69]: If you do a clean pod reinstall, I don't think both syntaxes will work.
The comment might lead to people reporting import issues.
Can you try `pod deintegrate && pod install`?
- if `import ZipArchive` works after that, then probably just leave "import ZipArchive" in the doc.
- If `import SSZipArchive` is required, the specify "import ZipArchive" for SPM/Carthage, and "import SSZipArchive"  for CocoaPods.

Same thing for ObjC.

Comment (jhudsonWA) [reply README.md:69]: You are right, I created a fresh project and pods does require SSZipArchive. I'll match your language.

Comment (jhudsonWA) [reply README.md]: Ok, language updated.

**Comment by jhudsonWA** [comment]
I'm going to merge in.

**Pull request merged**